AEPEdgeIdentity.xcodeproj/project.pbxproj

```diff
@@ -891,6 +891,7 @@
 		};
 		D43BAC2025BF4A0C00B08CC0 /* ShellScript */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
make sure you uncheck the "based on dependency analysis" box in Swift lint action for all xcode targets that use it